### PR TITLE
NIFI-11194 Remove gzip CSS and JS from nifi-web-ui

### PR DESCRIPTION
**Pull request by exceptionfactory**
# Summary

[NIFI-11194](https://issues.apache.org/jira/browse/NIFI-11194) Removes unnecessary gzip processing for CSS and JavaScript files in `nifi-web-ui`, avoiding unnecessary building and packaging steps.

# Tracking

Please complete the following tracking steps prior to pull request creation.

### Issue Tracking

- [X] [Apache NiFi Jira](https://issues.apache.org/jira/browse/NIFI) issue created

### Pull Request Tracking

- [X] Pull Request title starts with Apache NiFi Jira issue number, such as `NIFI-00000`
- [X] Pull Request commit message starts with Apache NiFi Jira issue number, as such `NIFI-00000`

### Pull Request Formatting

- [X] Pull Request based on current revision of the `main` branch
- [X] Pull Request refers to a feature branch with one commit containing changes

# Verification

Please indicate the verification steps performed prior to pull request creation.

### Build

- [X] Build completed using `mvn clean install -P contrib-check`
  - [X] JDK 11
  - [ ] JDK 17

### Licensing

- [ ] New dependencies are compatible with the [Apache License 2.0](https://apache.org/licenses/LICENSE-2.0) according to the [License Policy](https://www.apache.org/legal/resolved.html)
- [ ] New dependencies are documented in applicable `LICENSE` and `NOTICE` files

### Documentation

- [ ] Documentation formatting appears as expected in rendered files

### CODE REVIEW

**Comment by exceptionfactory** [comment]
The Gzip files provided Jetty with the option to stream files that were already compressed, but this feature is deprecated and disabled by default in Jetty 9.4 as described in [Jetty PR 2385](https://github.com/eclipse/jetty.project/pull/2385).

**Comment by mcgilman** [comment]
The changes look good. Will review closer and verify...

**Review by mcgilman — APPROVED** [review]
Thanks @exceptionfactory

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/pom.xml [hunk 1]
```diff
@@ -896,81 +896,45 @@
                             <packagingIncludes>
                                 assets/**/*,
                                 css/common-ui.css,
-                                css/common-ui.css.gz,
```
[the remark this answers is not in the document]
Comment (exceptionfactory) [reply nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/pom.xml:899]: Thanks @mcgilman, I pushed a commit updating that comment as suggested.

**Pull request merged**